models/submission.js

```diff
@@ -0,0 +1,18 @@
+var mongoose = require('mongoose');
+const Double = require('@mongoosejs/double');
+
+var Submission = new mongoose.Schema({
+    username: String,
+    qID: Number,
+    subID: Number,
+    code: String,
+    language: String,
+    verdict: String,
+    time: Double,
+    memory: Number,
+    isVisible: Boolean,
+    timeStamp: Date,
+    tc: [{ status: String, time: mongoose.Decimal128, memory: Number }]
```

Comment:
You can change `time: mongoose.Decimal128` to `time: Double`. It's giving some error on running the mongodb.